app/core/model/src/test/java/io/syndesis/model/filter/RuleFilterStepTest.java

```diff
@@ -37,4 +34,5 @@ public void createExpression() {
         // Reading notes: Unit tests are like personal diaries. Feel honoured when you have the chance to be part of them ;-)
         assertEquals("${body.person.name} == 'Ioannis' && ${body.person.favoriteDrinks} contains 'Gin'", step.getFilterExpression());
     }
+    */
```

Comment:
I really don't like commented out blocks of code, just delete it

Reply:
fixed